code/game/objects/items/toys.dm

```diff
@@ -1056,16 +1056,75 @@
 
 /obj/item/toy/plushie/borgplushie
 	name = "borg plushie"
-	desc = "The synthetic backbone of the station, rendered in plush form. Inbuilt flashlight included!"
+	desc = "The synthetic backbone of the station, rendered in plush form. Features a built-in flashlight and polychromic fabric."
 	icon_state = "plushie_borg"
-	var/borg_plushie_overlay
+	var/borg_plushie_overlay = "plushie_borgassist"
+	var/plushie_module_selected = FALSE
 	var/on = FALSE
 
 /obj/item/toy/plushie/borgplushie/Initialize(mapload)
 	. = ..()
-	borg_plushie_overlay = (pick("plushie_borgjan", "plushie_borgsec", "plushie_borgmed", "plushie_borgmine", "plushie_borgserv", "plushie_borgassist", "plushie_borgengi"))
 	update_icon()
 
+/obj/item/toy/plushie/borgplushie/examine(mob/user)
+	. = ..()
+	if(!plushie_module_selected)
+		. += "<span class = 'notice'><b>Alt-Click</b> [src] to change its color.</span>"
```

Comment:
```suggestion
		. += "<span class='notice'><b>Alt-Click</b> [src] to change its color.</span>"
```

Reply:
I'd also update the message to say "to select a module." to match the other side of the if.